fe/src/App.vue

```diff
@@ -1,23 +1,20 @@
 <template>
   <v-app>
     <v-main>
-      <HelloWorld />
+      <router-view/>
+      <!-- <RegistrationView/> -->
     </v-main>
   </v-app>
 </template>
 
 <script>
-import HelloWorld from "./pages/HelloWorld.vue";
+  // import RegistrationView from './pages/RegistrationView.vue';
```

Comment:
@LennardUy104 Please avoid unnecessary comments, please confirm if this is still needed. If not, please remove.

Reply:
@roseaugusto Already resolve changes